src/tests/main.test.ts

```diff
@@ -222,6 +222,36 @@ describe("Purchases.generateRevenueCatAnonymousAppUserId()", () => {
     const anonymousId = Purchases.generateRevenueCatAnonymousAppUserId();
     expect(() => Purchases.configure(testApiKey, anonymousId)).not.toThrow();
   });
+
+  test("works with crypto.randomUUID", () => {
+    const randomUUIDSpy = vi.spyOn(crypto, "randomUUID");
+
+    const anonymousId = Purchases.generateRevenueCatAnonymousAppUserId();
+    expect(anonymousId).toMatch(/^\$RCAnonymousID:[0-9a-f]{32}$/);
+    expect(randomUUIDSpy).toHaveBeenCalled();
+  });
+
+  test("falls back to crypto.getRandomValues when randomUUID is not available", () => {
+    crypto.randomUUID = undefined as unknown as typeof crypto.randomUUID;
+
+    const getRandomValuesSpy = vi.spyOn(crypto, "getRandomValues");
+
+    const anonymousId = Purchases.generateRevenueCatAnonymousAppUserId();
+    expect(anonymousId).toMatch(/^\$RCAnonymousID:[0-9a-f]{32}$/);
+    expect(getRandomValuesSpy).toHaveBeenCalled();
+  });
+
+  test("falls back to Math.random when crypto methods are not available", () => {
```

Comment:
Hmm would it be worth adding some tests for each of these cases that calling the method multiple times, it generates different UUIDs? NABD though.

Reply:
Addressed